packages/manager-react-components/src/hooks/datagrid/useResourcesV6.tsx

```diff
@@ -6,21 +6,21 @@ import {
   applyFilters,
   FilterTypeCategories,
   FilterComparator,
+  Filter,
 } from '@ovh-ux/manager-core-api';
-import { useQuery } from '@tanstack/react-query';
-import { useColumnFilters, ColumnSort } from '../../components';
+import { Query, useQuery } from '@tanstack/react-query';
+import { DatagridColumn, useColumnFilters, ColumnSort } from '../../components';
 
-export interface ColumnDatagrid {
-  cell: (props: any) => React.JSX.Element;
-  id: string;
-  label: string;
-  type: string;
-  isSearchable?: boolean;
-}
+export type ApiData<T> = {
```

Comment:
You could use the `IcebergFetchResultV6<T>` instead

Reply:
Done

Reply:
Actually sorry I didn't understood that you wanted to support non iceberg calls, so I was suggesting to use the type directly provided by core api but it's not relevant